doc/developer-documentation/aeproject-cli/tx-inspect.md

```diff
@@ -0,0 +1,43 @@
+# aeproject inspect
+
+## Inspect raw tx 
+The **inspect** command unpack and verify raw transaction (verify nonce, ttl, fee, account balance).
```

Comment:
adding something like "it can be used to verify failing transactions to check what the issue is", to emphasize that its main purpose is for failing transactions.